### ISSUE TITLE
Built in map creator

### ISSUE THREAD

**Issue by alikeobservant** [studio-minus/ppg-bugs-and-requests#202]
**Map creator**
Button on the main menu that says "Map editor." You can click on this and it will bring you to a template sort of void map. You can spawn in cubes with different colors, textures, collisions, and maybe noise emitting cubes? you can enable them to be a physics object, emit light, everything a map creator needs. You can customize the map however you want. Preferably maybe like a side menu that just says "spawn - delete." when you spawn a cube you can edit its properties. Something sort of how like you'd build in normal ppg maps, but everything is frozen, there are a lot more options (which are now easier to use,  so e.g a light-emitting cube instead of putting a box and a lamp inside) and HOPEFULLY water spawning options. Once you are done designing your map you can playtest it, and then upload it to the workshop with a new "Map" tag.

tl;dr make map editor
**Feature purpose**
map making go brrrr

**Comment by TimSvenson** [comment]
this would be pog

**Comment by WWWOWWWPL** [comment]
bruv mestiez is already working on a map editor

**Comment by DindinYT37** [comment]
Don't tell anyone but, that's already in the works.

Source: Trust me bro

**Comment by mestiez** [comment]
This is already being worked on
